src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n + 1) {
```

Comment:
The space between `.stars--1` and `:nth-child(-n + 1)` might cause issues. This space indicates a descendant selector, which means it will select any `:nth-child(-n + 1)` within `.stars--1`. If the intention is to select direct children, remove the space.